### ISSUE TITLE
ClassCastException in EntityDamageByEntity event

### ISSUE THREAD

**Issue by sirsavary** [BeYkeRYkt/LightSource#1]
Here is the full stack trace:

```
[14:28:22] [Server thread/ERROR]: Could not pass event EntityDamageByEntityEvent to LightSource v2.1.5
org.bukkit.event.EventException
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.craftbukkit.v1_8_R1.event.CraftEventFactory.callEvent(CraftEventFactory.java:85) [spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.craftbukkit.v1_8_R1.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:538) [spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.craftbukkit.v1_8_R1.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:451) [spigot.jar:git-Spigot-320ed92-01f0122]
    at org.bukkit.craftbukkit.v1_8_R1.event.CraftEventFactory.handleNonLivingEntityDamageEvent(CraftEventFactory.java:589) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.EntityItemFrame.damageEntity(EntityItemFrame.java:31) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.EntityHanging.l(EntityHanging.java:163) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.EntityHuman.attack(EntityHuman.java:959) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.EntityPlayer.attack(EntityPlayer.java:1047) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.PlayerConnection.a(PlayerConnection.java:1312) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.PacketPlayInUseEntity.a(SourceFile:52) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.PacketPlayInUseEntity.a(SourceFile:11) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.PacketHandleTask.run(SourceFile:13) [spigot.jar:git-Spigot-320ed92-01f0122]
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_31]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_31]
    at net.minecraft.server.v1_8_R1.MinecraftServer.z(MinecraftServer.java:683) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.DedicatedServer.z(DedicatedServer.java:316) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.MinecraftServer.y(MinecraftServer.java:623) [spigot.jar:git-Spigot-320ed92-01f0122]
    at net.minecraft.server.v1_8_R1.MinecraftServer.run(MinecraftServer.java:526) [spigot.jar:git-Spigot-320ed92-01f0122]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_31]
Caused by: java.lang.ClassCastException: org.bukkit.craftbukkit.v1_8_R1.entity.CraftItemFrame cannot be cast to org.bukkit.entity.LivingEntity
    at ykt.BeYkeRYkt.LightSource.LightListener.onEntityDamage(LightListener.java:341) ~[?:?]
    at sun.reflect.GeneratedMethodAccessor233.invoke(Unknown Source) ~[?:?]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_31]
    at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_31]
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:301) ~[spigot.jar:git-Spigot-320ed92-01f0122]
    ... 22 more
```

**Comment by BeYkeRYkt** [comment]
FIXED 2.1.6

**Comment by sirsavary** [comment]
Great, thank you.
